apps/core/src/pages/privacy-policy/index.tsx

```diff
@@ -0,0 +1,370 @@
+    display: "flex",
+    flexDirection: "column",
+    alignItems: "center",
+    borderRight: `1px solid ${
+      theme.colorScheme === "dark" ? theme.colors.dark[7] : theme.colors.gray[3]
+    }`
+  },
+
+  main: {
+    flex: 1,
+    backgroundColor:
+      theme.colorScheme === "dark"
+        ? theme.colors.dark[6]
+        : theme.colors.secondary[4],
+    padding: 0
+  },
+
+  mainLink: {
+    width: 44,
+    height: 44,
+    borderRadius: theme.radius.md,
+    display: "flex",
+    alignItems: "center",
+    justifyContent: "center",
+    color:
+      theme.colorScheme === "dark"
+        ? theme.colors.dark[0]
+        : theme.colors.gray[7],
+
+    "&:hover": {
+      backgroundColor:
+        theme.colorScheme === "dark"
+          ? theme.colors.dark[5]
+          : theme.colors.gray[0]
+    }
+  },
+
+  mainLinkActive: {
+    "&, &:hover": {
+      backgroundColor: theme.fn.variant({
+        variant: "light",
+        color: theme.primaryColor
+      }).background,
+      color: theme.fn.variant({ variant: "light", color: theme.primaryColor })
+        .color
+    }
+  },
+
+  title: {
+    boxSizing: "border-box",
+    fontFamily: `Greycliff CF, ${theme.fontFamily}`,
+    marginBottom: theme.spacing.xl,
+    backgroundColor:
+      theme.colorScheme === "dark"
+        ? theme.colors.dark[7]
+        : theme.colors.secondary[4],
+    padding: theme.spacing.md,
+    paddingTop: 18,
+    height: 60,
+    borderBottom: `1px solid ${
+      theme.colorScheme === "dark" ? theme.colors.dark[7] : theme.colors.gray[3]
+    }`
+  },
+
+  logo: {
+    boxSizing: "border-box",
+    width: "100%",
+    display: "flex",
+    justifyContent: "center",
+    height: 60,
+    paddingTop: theme.spacing.md,
+    borderBottom: `1px solid ${
+      theme.colorScheme === "dark" ? theme.colors.dark[7] : theme.colors.gray[3]
+    }`,
+    marginBottom: theme.spacing.xl
+  },
+
+  link: {
+    boxSizing: "border-box",
+    display: "flex",
+    alignItems: "center",
+    textDecoration: "none",
+    borderTopRightRadius: theme.radius.md,
+    borderBottomRightRadius: theme.radius.md,
+    color:
+      theme.colorScheme === "dark"
+        ? theme.colors.dark[0]
+        : theme.colors.gray[7],
+    padding: `0 ${theme.spacing.md}px`,
+    fontSize: theme.fontSizes.sm,
+    marginRight: theme.spacing.md,
+    fontWeight: 500,
+    height: 44,
+
+    "&:hover": {
+      backgroundColor:
+        theme.colorScheme === "dark"
+          ? theme.colors.dark[5]
+          : theme.colors.gray[1],
+      color: theme.colorScheme === "dark" ? theme.white : theme.black
+    }
+  },
+
+  linkActive: {
+    "&, &:hover": {
+      borderLeftColor: theme.fn.variant({
+        variant: "filled",
+        color: theme.primaryColor
+      }).background,
+      backgroundColor: theme.fn.variant({
+        variant: "filled",
+        color: theme.primaryColor
+      }).background,
+      color: theme.white
+    }
+  }
+}));
+
+// breadcrumbs
+const items = [
```

Comment:
name the item breadcrumbs instead of having to add a comment saying that's what it is